src/classes/CON_ContactMerge_TDTM.cls

```diff
@@ -77,7 +77,7 @@ public class CON_ContactMerge_TDTM extends TDTM_Runnable {
                 mergeKeys.add(generateMergeKey(oldContact.Id, oldContact.AccountId, oldContact.MasterRecordId));
             }
 
-            handleContactMergeFixupsFuture(mergeKeys);
+            System.enqueueJob(new ContactMergeFixupQueueable(mergeKeys));
```

Comment:
Though I'm pretty sure that Customizable Rollups run synchronously when called from this class, it's worth testing  this with CRLP both enabled and disabled to ensure that the rollups run as expected without any errors.

Reply:
Worse case, it is feasible to change queueables so that if needed ContactMergeFixupQueueable() could execute another queueable to handle the Contact rollups.